docs/Clearwater_Stateful_Statistics.md

```diff
@@ -25,6 +25,26 @@ To then calculate the number of registrations actually active in the deployment,
 
 Note: These statistics may be inaccurate in systems with a failing node. As such they should be considered unreliable while any alarms are raised, particularly those relating to Chronos.
 
+For a deployment which contains more that one site, the statistics must be summed across all sites in the deployment, as by default Chronos is not geographically redundant.
```

Comment:
Should these docs include what to do if Chronos GR is enabled? I think this could be simply removing ', as by default Chronos is not geographically redundant.' from the first sentence, then at the end of this section adding something like 'If Chronos replicates its timers to the other sites (which it doesn't do by default), then you should divide your result by the number of sites'.

Reply:
done